target/linux/ath79/image/generic.mk

```diff
@@ -75,6 +75,14 @@ define Device/seama
   SEAMA_SIGNATURE :=
 endef
 
+define Device/8dev_carambola2
+  ATH_SOC := ar9331
+  DEVICE_TITLE := 8devices Carambola2
```

Comment:
You should add `SUPPORTED_DEVICES += carambola2` for the seamless upgrades from ar71xx.